addons/pos_restaurant/static/src/overrides/models/pos_store.js

```diff
@@ -319,7 +319,7 @@ patch(PosStore.prototype, {
         );
     },
     tableHasOrders(table) {
-        return Boolean(table.getOrder());
+        return table.getOrder();
```

Comment:
```suggestion
        return Boolean(table.getOrder());
```

Restore this diff. I think only the change is `getOrder` is needed.